### PR TITLE
webedit stimpaks into freezers and vr workbenches also mob spawners

### PR DESCRIPTION
**Pull request by buffyuwu**
## About The Pull Request
adds animated water effects when entering water
arcade sounds no longer play at 100 volume because dear god why was this a thing
also fixes some annoying bos bunker things like a bench under a chair and a floating camera
fixes missing icon for mob spawners as well as a weird runtime protectrons and handys had when they get destroyed by fixing gib() itself. also increases the probability their body will decompose.
reduces number of spawners on the surface

## Pre-Merge Checklist
- [x] Your Pull Request contains no breaking changes
- [x] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
:cl:
tweak: you can now store stimpaks in chemical storage fridges like any other medicine
fix: you can now put materials into the VR workbenches and lathes
tweak: elevators no longer gib you
fix: fixed missing icons for mob spawners and increased their odds to decompose
tweak: reduced overall instanced mob count
/:cl:

### CODE REVIEW

File: code/modules/shuttle/on_move.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@ All ShuttleMove procs go here
 				M.stop_pulling()
 				M.visible_message("<span class='warning'>[shuttle] slams into [M]!</span>")
 				SSblackbox.record_feedback("tally", "shuttle_gib", 1, M.type)
-				M.gib()
+				M.adjustBruteLoss(120)
```
Comment (printer16) [inline code/modules/shuttle/on_move.dm:36]: undocumented change

**Review by hecksadecimal — CHANGES_REQUESTED** [review]
Use the PR template.

**Review by Looreydev — CHANGES_REQUESTED** [review]
Please use the Pull Request Template, the full one.

**Review by hecksadecimal — DISMISSED** [review]

File: code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm [hunk 2]
```diff
@@ -468,6 +468,7 @@
 					/obj/item/reagent_containers/glass/bottle,
 					/obj/item/reagent_containers/glass/beaker,
 					/obj/item/reagent_containers/spray,
+					/obj/item/reagent_containers/hypospray,
 					// /obj/item/reagent_containers/medigel,
```
Comment (Looreydev) [inline code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm:472]:
```suggestion
```

Comment (Looreydev) [reply code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm:472]: While you're at editing the file, be so nice and remove that comment.

**Review by Looreydev — CHANGES_REQUESTED** [review]

**Review by Looreydev — APPROVED** [review]

**Pull request merged**